pkg/webhook/namespacelabel.go

```diff
@@ -80,3 +82,12 @@ func matchesPrefix(s string) bool {
 	}
 	return false
 }
+
+func matchesSuffix(s string) bool {
```

Comment:
Can you please add a test for this like https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/webhook/namespacelabel_test.go#L165